launcher/src/main/java/org/apache/spark/launcher/Main.java

```diff
@@ -101,6 +99,22 @@ public static void main(String[] argsArray) throws Exception {
     }
   }
 
+  /**
+   * Prepare spark commands with the appropriate command builder.
+   * If printLaunchCommand is set then the commands will be printed to the stderr.
+   */
+  private static List<String> buildCommand(AbstractCommandBuilder builder,
```

Comment:
Please follow the Spark style for multi-line parameter lists.

Reply:
Fixed.

Just for the sake of my understanding what does `many` mean in the following? All?
`For Java code, Apache Spark follows Oracle’s Java code conventions. Many Scala guidelines below also apply to Java.`

Reply:
Read that as "unless it cannot apply to Java at all, follow the Scala style".